fbw-common/src/wasm/terronnd/build.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
Is it possible to add lto to the compilation or does this break stuff? Because everything else is compiled with lto.

Reply:
I am not a fan of LTO. Doesn't improve the performance so much and can break stuff with certain optimization levels if the code is not warning free. And our external code is not warning free

Reply:
You have a point there.